3rd_party/requests/README.md

```diff
@@ -0,0 +1,29 @@
+# Requests -library
+
+Requests -library is one of the most known and used Python libraries.
+
+👉 Their slogan tells it all: `HTTP for Humans`
+
+Whether you need to download a file or check if a website is responding, for everything HTTP, you should just use `requests.`
+
+## Usage
+
+```python
+import requests
+
+def download_file(url, local_filename):
+    response = requests.get(url)
+    response.raise_for_status()  # This will raise an exception if the request fails
+    with open(local_filename, 'wb') as f:
+        f.write(response.content)  # Write the content of the response to a file
+    return local_filename
+```
+
+Examples in Robocorp Portal:
```

Comment:
Right before this "Examples" line, I would also add an entry called "Snippets", where we reference every snippet we built, so we can list them all along with a description. (and for also highlighting the `[snippets](./snippets)` folder itself)